mm2src/coins/utxo/utxo_standard.rs

```diff
@@ -1246,7 +1246,17 @@ impl HDWalletBalanceOps for UtxoStandardCoin {
         &self,
         addresses: Vec<Address>,
     ) -> BalanceResult<Vec<(Address, Self::BalanceObject)>> {
-        utxo_common::addresses_balances(self, addresses).await
+        let ticker = self.ticker().to_string();
+        let balances = utxo_common::addresses_balances(self, addresses).await?;
+
+        balances
+            .into_iter()
+            .map(|(address, balance)| {
+                let mut balance_by_ticker = HashMap::with_capacity(1);
+                balance_by_ticker.insert(ticker.clone(), balance);
+                Ok((address, balance_by_ticker))
```

Comment:
nit: `Ok((address, HashMap::from([(ticker.clone(), balance)])))`

Reply:
Using `with_capacity(1)` might be better for this case due to direct allocation and insertion without iterator/array overhead.

Reply:
not really a big deal but i think it's more concise. for the iterator though, it will have size hints since its size is computable and thus the hashmap shouldn't over allocate.

Reply:
Will do `Ok((address, HashMap::from([(ticker.clone(), balance)])))` for consistency with other implementations and the overhead is negligible if there is actually one.

Reply:
Done